tests/mocks/HUBComponentMock.h

```diff
@@ -96,6 +102,9 @@ NS_ASSUME_NONNULL_BEGIN
 /// Whether the component should act like it is a content offset observer or not
 @property (nonatomic) BOOL isContentOffsetObserver;
 
+/// Whether the component should act like it is a component action observer or not
+@property (nonatomic) BOOL isComponentActionObserver;
```

Comment:
Nitpick, we already know it's a component, so no need to include it in the property name. `isActionObserver` is enough, and also more consistent with the rest of the API 🙂